README.md

```diff
@@ -1,2 +1,48 @@
-# BOCS Control
-Control scripts for the BOCS.
+# BOCS Data Collection
+
+Contains the source code for the `python3-bocs` Debian package that runs the BOCS air quality instrument, recording data to local storage.
+Released Deb files are available at <TODO>.
+
+# Setup Pi
+
+The RaspberryPi needs setting up before the BOCS software can be installed.
+
+  1. Install the latest version of [RaspberryPi OS with desktop](https://www.raspberrypi.com/software/) to a microSD card
+  2. Boot up the Pi (outside of the BOCS) and follow the onscreen instructions to setup locale, password, WiFi access (if needed), and **make sure to update the software**
+  3. Reboot
+  4. Run `sudo raspi-config`, go into Interface Options and enable both `SSH` and `Legacy Camera`
+  5. Reboot
+
+The Pi can now be inserted into the BOCS and interfaced with by either SSH or connecting a keyboard to the USB port on the front panel.
+
+# Installation of BOCS data collection software
+
+With the Pi housed in the BOCS and all wires connected as shown in <TODO document>, the data collection software is ready to be run.
```

Comment:
I don't think we need to reference the connection / build instructions here - this is just a document to describe how to install the BOCS software on the Pi. It doesn't need to be in the BOCS at all to install the package.

Reply:
Fair